test/local/devices.js

```diff
@@ -208,43 +208,59 @@ describe('devices', () => {
         uaBrowser: 'foo',
         uaBrowserVersion: 'bar',
         uaOS: 'baz',
-        uaOSVersion: 'qux'
-      }), 'foo bar, baz qux', 'result is correct when all ua properties are set')
+        uaOSVersion: 'qux',
```

Comment:
I'm pretty comfortable with the coverage here now fwiw, not sure adding more tests will achieve anything.